packages/devtools_app/integration_test/test/live_connection/app_test.dart

```diff
@@ -5,14 +5,14 @@
 import 'dart:ui' as ui;
 
 import 'package:devtools_app/devtools_app.dart';
-import 'package:devtools_app/src/framework/landing_screen.dart';
-import 'package:devtools_app/src/framework/release_notes/release_notes.dart';
 import 'package:devtools_app/src/shared/primitives/simple_items.dart';
 import 'package:devtools_test/devtools_integration_test.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:integration_test/integration_test.dart';
 
+import '../../test_infra/utils/connection.dart';
```

Comment:
we already have an `integration_test_utils.dart` filed in `packages/devtools_test`

we should consolidate. I can't remember why the integration_test_utils.dart file originally needed to be in `package:devtools_test`, but if it can be moved into `devtools_app/integration_test`, let's do that.

Reply:
Tests fail because they do not see '../../test_infra/utils/connection.dart'. Moved to devtools_test.